include/lsst/afw/geom/Transform.h

```diff
@@ -271,6 +278,15 @@ class Transform final {
      */
     explicit Transform(std::shared_ptr<ast::FrameSet> frameSet);
 
+    // implement afw::table::io::Persistable interface
+    std::string getPersistenceName() const override { return getShortClassName(); }
+
+    // implement afw::table::io::Persistable interface
+    std::string getPythonModule() const override { return "lsst.afw.geom"; }
```

Comment:
Should `SkyWcs` return "lsst.afw.geom" instead of "lsst.afw.geom.skyWcs"?

Reply:
Yes.  Before now I hadn't thought that distinction would matter, but now that I think about it, exposing the full implementation module here makes the persistence fragile to the kind of refactoring we want to do to shrink the pybind11 build sizes in the future.

Reply:
Thanks. I will fix the module name in SkyWcs.cc on tickets/DM-10765